### ISSUE TITLE
Using JSON instead of YAML

### ISSUE THREAD

**Issue by certik** [asdf-format/asdf-standard#124]
http://yaml.org/ lists only 3 libraries in C/C++/Fortran that can read it, one of the link (Syck) doesn't work, so there are only 2 libraries.

http://www.json.org/ lists 16 libraries in C, 19 in C++ and even 2 in Fortran, total of 37.

One thing that I don't like about HDF5 is that there is only one library that can read it, and it's pretty big. It would be nice if the ASDF format had many libraries that could read/write it. So JSON seems like a much better choice from this perspective.

Also JSON I think is simpler than YAML, so it's easier to write a parser.

**Comment by justincely** [comment]
Thanks for the info and thoughts, let's expand on this discussion when the venue discussed #123 is up!

**Comment by certik** [comment]
@justincely definitely. Then we can just post a conclusion in this issue. Please ping me once the venue is setup.

**Comment by braingram** [comment]
Duplicate of https://github.com/asdf-format/asdf-standard/issues/100
